### PR TITLE
* Enable .NET 9

### PR DESCRIPTION
**Pull request by Wagnerp**
* Enable .NET 9

image

### CODE REVIEW

File: Source/Krypton Components/Directory.Build.targets [hunk 1]
```diff
@@ -139,33 +139,33 @@
...
+				<Description Condition="'$(TFMs)' == 'all'">An update to Component factory's krypton toolkit to support .NET Framework 4.6.2 - 4.8.1 and .NET 6 - 9. $(Description) This package supports all .NET Framework versions starting .NET Framework 4.6.2 - 4.8.1 and .NET 6 - 7. Also, all libraries are included targeting each specific framework version for performance purposes. To view all of the standard toolkit package latest version information, please visit: https://github.com/Krypton-Suite/Krypton-Toolkit-Suite-Version-Dashboard/blob/main/Documents/Modules/Standard/Krypton-Toolkit-Suite-Standard-Modules.md To find out what's new, please visit: https://github.com/Krypton-Suite/Standard-Toolkit/blob/canary/Documents/Help/Changelog.md This package is for those who want to try out the latest features before deployment.</Description>
```
Comment (Smurf-IV) [inline Source/Krypton Components/Directory.Build.targets:142]: Please read through as it still has `.Net 6 - 7` in the text..

File: Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj [hunk 2]
```diff
@@ -39,6 +39,7 @@
         <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
         <Configurations>Debug;Release;Installer;Nightly;Canary</Configurations>
         <LangVersion>preview</LangVersion>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj:42]: Why ?

File: Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj [hunk 3]
```diff
@@ -3,19 +3,19 @@
...
                 <When Condition="'$(Configuration)' == 'Canary'">
                     <PropertyGroup>
-                        <TargetFrameworks>net462;net47;net471;net472;net48;net481;net6.0-windows;net7.0-windows;net8.0-windows</TargetFrameworks>
+                        <TargetFrameworks>net462;net47;net471;net472;net48;net481;net6.0-windows;net7.0-windows;net8.0-windows;net9.0-windows</TargetFrameworks>
                     </PropertyGroup>
                 </When>
                 <Otherwise>
                     <PropertyGroup>
-                        <TargetFrameworks>net48;net481;net6.0-windows;net7.0-windows</TargetFrameworks>
+                        <TargetFrameworks>net48;net481;net8.0-windows;net9.0-windows</TargetFrameworks>
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj:18]: Please keep net 6 as it is LTS

File: Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs [hunk 4]
```diff
@@ -4556,8 +4556,52 @@ internal struct TITLEBARINFOEX
...
+        // https://withinrafael.com/2015/07/08/adding-the-aero-glass-blur-to-your-windows-10-apps/
+        // https://github.com/riverar/sample-win32-acrylicblur/blob/917adc277c7258307799327d12262ebd47fd0308/MainWindow.xaml.cs
+
+        [DllImport(Libraries.User32)]
+        public static extern int SetWindowCompositionAttribute(HandleRef hWnd, in WindowCompositionAttributeData data);
+
+        public unsafe struct WindowCompositionAttributeData
+        {
+            public WCA Attribute;
+            public void* Data;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs:4573]: This should be an `IntPtr` and therefore no unsafe code.
Please **remove** the acrylic work from this PR

File: Source/Krypton Components/Krypton.Toolkit/General/ShadowManager.cs [hunk 5]
```diff
@@ -7,9 +7,7 @@
  */
 #endregion
 
-// To get around bug in .NET 8, Preview 7
-// TODO: Remove when .NET 8 is GA
-#if NET8_0
+#if NET8_0_OR_GREATER
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/ShadowManager.cs:10]: Was the `bug` not fixed, or is this just a blind change ?

File: Source/Krypton Components/Krypton.Toolkit/Utilities/WindowUtilities.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Utilities/WindowUtilities.cs]: - Add License
- Remove usings
- Remove as part of _Acrylic_ changeset!

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Outlook Grid/VisualCustomFormatRuleRtlAwareForm.cs [hunk 7]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Outlook Grid/VisualCustomFormatRuleRtlAwareForm.cs:1]: - Add License
- Remove usings

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/General/ShadowManager.cs (hunk 5 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/ShadowManager.cs:10]: No, there must have had been changes between .NET 7 & 8 in `MethodInvoker`. Removing will cause the build to fail.

File: Source/Krypton Components/Directory.Build.targets (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Directory.Build.targets:142]: Done

File: Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj:42]: Removed, was for Acrylic code, now commented out

File: Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs (hunk 4 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs:4573]: Commented out ready to go back to later

**Pull request merged**